versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/logic/IndexesLogicImpl.java

```diff
@@ -550,17 +553,24 @@ void completeIndexesInCommitChain(
 
       commitOperations(current).forEach(newIndex::add);
 
-      CommitObj newCommit =
+      Builder c =
```

Comment:
Let's qualify the `Builder` as `CommitObj.Builder` (otherwise checkstyle should also complain).